dist/css/uw-ui-toolkit.css

```diff
@@ -6588,7 +6587,7 @@ To use your own licensed copy of FrizQuadrata, un comment the @font-face definit
   background-color: #b70101;
   color: #fff;
   box-shadow: 0px 2px 0px #600;
-  border: 0px solid transparent;
+  border: 1px solid #b70101;
```

Comment:
Yea. I think I'm going to go through all the colors and use variables so I can do it then. It'd be nice to have it all in one commit/PR.